robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityWindowInfo.java.vm

```diff
@@ -96,6 +144,24 @@ public class ShadowAccessibilityWindowInfo {
   public void setId(int value) {
     id = value;
   }
+
+  public void setLayer(int value) {
+    layer = value;
+  }
+
+  public void setFocused(boolean focused) {
+    isFocused = focused;
+  }
+
+  public void addChild(AccessibilityWindowInfo child) {
+    if (children == null) {
+      children = new LinkedList<>();
+    }
+
+    children.add(child);
+    ((ShadowAccessibilityWindowInfo) ShadowExtractor.extract(child)).parent =
+        mRealAccessibilityWindowInfo;
+  }
```

Comment:
I believe we're missing three methods: equals, hashcode, and obtain(AccessibilityWindowInfo).